src/domain/navigation/index.ts

```diff
@@ -176,7 +176,7 @@ export function parseUrlPath(urlPath: string, currentNavPath: Path<SegmentType>,
         } else if (type === "details" || type === "members") {
             pushRightPanelSegment(segments, type);
         } else if (type === "member") {
-            const userId = iterator.next().value;
+            const userId = decodeURIComponent(iterator.next().value);
```

Comment:
`userId` can be `undefined` here, which `decodeURIComponent` helpfully converts into the truthy `"undefined"` value, so should convert after this check.